PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java

```diff
@@ -17,21 +17,21 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import com.coffeelovers.pcu.model.RawMaterial;
-import com.coffeelovers.pcu.model.RawMaterialRepository;
+import com.coffeelovers.pcu.model.InvoiceDetails;
+import com.coffeelovers.pcu.model.InvoiceDetailsRepository;
 
 @CrossOrigin(origins = "http://localhost:8081")
 @RestController
 @RequestMapping("/api")
 public class RawMaterialController {
 	
 	@Autowired
-	RawMaterialRepository rawMaterialRepository;
+	InvoiceDetailsRepository rawMaterialRepository;
 	
 	@GetMapping("/rawMaterials")
```

Comment:
Should we also change the api?
```suggestion
	@GetMapping("/invoiceDetails")
```

Reply:
No need. it is already done on my side. ignore it please